### PR TITLE
1.2 dependency upgrade

### PR DESCRIPTION
**Pull request by i386**
# Description

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -301,18 +301,24 @@
...
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.13</version>
+            <version>2.16</version>
         </dependency>
+        
+        <!-- 
+            workflow-job >= 2.12 requires a Jenkins core >= 2.60.
+            Do not upgrade this dependency without a discussion on jenkinsci-ux@googlegroups.com
+        -->
```
Comment (i386) [inline pom.xml]: @michaelneale adding a note about the dependency here.

Comment (vivek) [inline pom.xml]: As tested with experimental update center, there might be beta version of other dependencies out there that might depend on it resulting in to blueocean install errors - specially if blueocean is installed on latest LTS (2.46.3) without installing required plugins.

**Comment by vivek** [comment]
@i386 how about workflow-step-api:1.10?

**Comment by i386** [comment]
@vivek I've updated this with more dependency upgrades

**Comment by i386** [comment]
@vivek passes ATH

**Pull request merged**